schemas/questions/types/general.json

```diff
@@ -35,9 +35,75 @@
         "type": "string",
         "enum": ["General"]
       },
+      "dynamic_answers": {
+        "type": "object",
+        "properties": {
+          "values": {
+            "type": "object",
+            "source": ["list"],
+            "identifier": {
+              "$ref": "../../common_definitions.json#/non_empty_string"
+            }
+          }
+        },
+        "answers": {
```

Comment:
We need to write some custom validation to validate answer source used inside "dynamic_answers.answers", either in placeholders or validation. Since answer sources by default uses the list item id of the item, any answer sources that references non-repeating answers will need a "list_item_selector" defined otherwise it will error. This needs to be validate and also tested on the runner side.

Reply:
This one is outdated as validation is not needed anymore since we made the relevant changes in runner.